src-graphql/base64.js

```diff
@@ -0,0 +1,7 @@
+export default function base64(str) {
+  if (typeof btoa === 'function') {
+    return btoa(str);
+  } else {
```

Comment:
We should check whether `Buffer` exists also. For instance I maybe using an un-shimmed version of the `js-buy-sk` in the Browser on an older browser. In this case we'd receive an error stating `Buffer` is not defined.

The else statement should guide the developer to use the shimmed version of the `js-buy-sdk`